src/components/Overview/StyleSelector.jsx

```diff
@@ -1,25 +1,36 @@
 import React, {useEffect, useState} from 'react';
 import {StyleImg} from './OverviewStyledCom.jsx';
+import {FaCheckCircle} from 'react-icons/fa';
 
 const StyleSelector = ({styles, currentStyle, setCurrentStyle}) => {
   return (
-  <div className='StyleSelector'>
-  {
-    currentStyle !== undefined
-    ? <h4>{`STYLE > ${currentStyle.name}`}</h4>
-    : <></>
-  }
-
-  {styles.map((style, i) => {
-    return <span key={style.style_id} >
-      {i % 4 === 0 && i !== 0
-      ? <br/>
-      : ' '}
-      <StyleImg style={{borderStyle: style === currentStyle ? 'solid' : ''}} src={style.photos[0].thumbnail_url} name={i} onClick={e => {setCurrentStyle(styles[e.target.name])}}/>
-    </span>
-  })}
-  </div>
+    <div className='StyleSelector'>
+    {
+      currentStyle !== undefined
```

Comment:
Here I believe you can shorten this up a bit:
can be {!currentStyle && [insert what you want visible]}
That way you don't have to provide a second option if you only want one thing to conditionally render.

Reply:
Could also be {currentStyle === undefined && [insert what element you want to be visible]}